modules/pam_xauth/pam_xauth.c

```diff
@@ -595,12 +614,12 @@ pam_sm_open_session (pam_handle_t *pamh, int flags UNUSED,
 						       cookiefile,
 						       "nlist",
 						       t,
-						       (unsigned long) getuid(),
-						       (unsigned long) getgid());
+						       (unsigned long) uid,
+						       (unsigned long) gid);
 					}
 					run_coprocess(pamh, NULL, &cookie,
-						      getuid(), getgid(),
-						      xauth, "-f", cookiefile,
+						      uid, gid,
+						      xauth, "-i", "-f", cookiefile,
```

Comment:
Why `-i` option is added here?  What it has to do with NFS and the cookiefile?

Reply:
see https://github.com/linux-pam/linux-pam/pull/556#issuecomment-1499245181 for the reason of the '-i' parameter